protos/orchestrator_service.proto

```diff
@@ -150,7 +150,7 @@ message EventRaisedEvent {
 }
 
 message GenericEvent {
-    string data = 1;
+    google.protobuf.StringValue data = 1;
```

Comment:
Do you know whether this could be a breaking change?

Reply:
TBH, I forget to consider whether it could be a breaking change. Thanks for bringing this up. 
In this case, I think I have to make sure the null value is acceptable for any logic use this event before making this update.

Reply:
As far as durable functions is concerned it won't be a breaking change. Server and client will always be upgraded together. Would it be an issue for dapr?

Reply:
So I checked dapr sdks, seems none of them have supported rewind yet. If the `GenericEvent` is only used in rewind logic, then it should not be a breaking change for dapr as well.

Reply:
I also don't think its a breaking change because [GenericEvent](https://github.com/Azure/durabletask/blob/6f902190fcf7f14d42b07e940f516e47525dbc18/src/DurableTask.Core/History/GenericEvent.cs#L22) in DurableTask.Core already allows `null`. It is odd/wrong for us to restrict `null` when sending across gRPC only.

Reply:
The bigger concern is with `durabletask-go`, which not only uses these protobufs as a wire format, but also as the storage format for serialized history events. A breaking change to the history event contract could cause an orchestration to fail to load if this were to result in an error. That said, if none of the gRPC-enabled SDKs today implement rewind, then we probably don't need to worry about whether this could be breaking.

Reply:
@cgillum ah that is good to know that go uses this for storage. This definitely could be a breaking change for it, but as you said it is unlikely it has used/stored this event yet.